numba/targets/arrayobj.py

```diff
@@ -1838,6 +1838,31 @@ def np_unique_impl(a):
     return np_unique_impl
 
 
+@overload(np.repeat)
+def np_repeat(a, repeats):
+    def np_repeat_impl(a, repeats):
+        a = a.ravel()
```

Comment:
Perhaps assign like `aravel = a.ravel()`, reassigning to the same variable name may cause confusion later?